query-feature-table/src/main/java/com/esri/arcgismaps/sample/queryfeaturetable/components/MapViewModel.kt

```diff
@@ -64,6 +61,17 @@ class MapViewModel(
         FeatureLayer.createWithFeatureTable(serviceFeatureTable)
     }
 
+    // map used to display the feature layer
+    val map = ArcGISMap(BasemapStyle.ArcGISTopographic)
+
+    private var usaViewpoint = Viewpoint(
+        center = Point(-11e6, 5e6, SpatialReference.webMercator()),
+        scale = 1e8
+    )
+
+    // define a MapViewpointOperation to set the initial viewpoint and update it when the viewpoint changes
+    var mapViewpointOperation: MapViewpointOperation? by mutableStateOf(null)
```

Comment:
The `mapViewpointOperation` here does not need to be nullable, you could set it to the initial viewpoint. And remove it from the init block. 

```suggestion
    // define a mutable MapViewpointOperation and set the initial viewpoint
    var mapViewpointOperation: MapViewpointOperation by mutableStateOf(
        MapViewpointOperation.Set(usaViewpoint)
    )
```